resources/dicts/patrols/general/medcat.json

```diff
@@ -279,5 +279,644 @@
+    ],
+    "intro_text": "r_c trots over to p_l, the two cats grinning at each other with raised tails. p_l wants an escort to gather herbs, and as luck would have it, the only cat free at the moment is their sibling, r_c!",
+    "decline_text": "... Or so they thought. r_c is called away by the deputy, and they rush off with an apologetic glance back, leaving p_l by themself with their tail sagging to the ground.",
+    "chance_of_success": 50,
+    "exp": 20,
+    "success_text": [
+        "p_l and r_c walk with their tails entwined and pelts brushing together, thrilled to be able to patrol together. p_l doesn't regret becoming a medicine cat, but they do feel secluded at times, watching r_c train and patrol with everyone. The siblings make the most of their time together, returning home side-by-side with their jaws full of herbs.",
+        "p_l and r_c race through the territory together, hearts light with joy at being able to patrol together despite their different roles. So what if they take a little longer coming home than they should? They still return to camp with plenty of herbs! They deserve to have some sibling time for a change.",
+        null,
+        "s_c seems a little distracted, deep in thought about something. When r_c asks what's up, s_c blinks slowly at them before confessing how much they miss spending time together. They burrow their noses into each others pelts and purr, soaking in the moment and their love for each other before continuing on, and promise to spend more time together. "
+    ],
+    "fail_text": [
+        "p_l and r_c are far too distracted catching up with one another, and get nothing useful done.",
+        "s_c seems a little distracted, deep in thought about something. When r_c asks what's up, s_c blinks slowly at them before confessing how much they miss spending time together. r_c tries to comfort them, but s_c shakes their head, knowing that no matter what they say, their roles are just too different. Both cats are too upset to gather herbs."
+    ],
+    "win_trait": [
+        "loving"
+    ],
+    "fail_trait": [
+        "lonesome",
+        "insecure"
+    ],
+    "relationship_constraint": [
+        "siblings",
+        "platonic_20"
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "antagonize_text": null,
+    "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_medwarriorapp_curiosity1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": [
+        "med_cat",
+        "warrior_apprentice",
+        "one_apprentice",
+        "random_herbs",
+        "respect",
+        "comfort",
+        "trust",
+        "platonic"
+    ],
+    "intro_text": "p_l leads app1 through c_n's territory, enjoying the nice day. They glance back, and realize app1 looks like they want to ask them a question.",
+    "decline_text": "Just as app1 opens their mouth to speak, p_l spots a Clanmate trying to remove a thorn from their paw over their shoulder, and they hurry away to help.",
```

Comment:
feels a lil awkward as the "p_l spots a Clanmate trying to remove a thorn from their paw over their shoulder" sounds less like p_l is spotting that over their shoulder and more like the Clanmate is somehow trying to bend over their own shoulder to remove a thorn.